Bottom: c982e1e5c2f37fca403544d2f79ceb515ecbcf4c
Top:    fbaa87034d6f8d1ee2c2a16ef8b4c331f4804be3
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-05 11:49:39 -0500

Report compilation summary

---

diff --git a/farm-compile b/farm-compile
index 72af274..b6c3e46 100755
--- a/farm-compile
+++ b/farm-compile
@@ -13,6 +13,7 @@ if test -d utils; then
 	target=driver
 fi
 
+error=
 for bintree in $farmbin/*; do
 	arch=
 	if test -r $bintree/configured-i386; then
@@ -34,7 +35,16 @@ for bintree in $farmbin/*; do
 		mv $log $log-ok
 		echo -e "\e[32mOK\e[0m"
 	else
+		error=1
 		mv $log $log-error
 		echo -e "\e[31mERROR\e[0m"
 	fi
 done
+
+if test -z "$error"; then
+		echo -e "\e[32mAll OK\e[0m"
+		exit 0
+	else
+		echo -e "\e[31mErrors found\e[0m"
+		exit 1
+fi
